build.gradle

```diff
@@ -17,12 +17,16 @@ subprojects {
     repositories {
         jcenter()
         maven { url "https://maven.google.com" }
+        flatDir {
+            dirs 'libs'
+        }
+        maven { url "https://dl.bintray.com/oktaappauth/okta-appauth" }
     }
 }
 
 ext {
-    versionNum = 1
-    versionName = "0.1.0"
+    versionNum = 2
```

Comment:
where is `versionNum` used? (just trying to understand why we have two version properties)

Reply:
Actually, we can ignore that one, just a habit )

Reply:
I would rather keep it for now and try to move it to some sort of autoincrementing. Currently, version code influences only application versioning, [os understands versions using versionNumber](https://developer.android.com/studio/publish/versioning). However, it doe not apply directly to libraries but it could be that process is going to be changed in the future. 
So I would recommend two things:
1. Keep it for now as it is
2. Change our version naming and version number increment by applying git based strategy.
VersionName- taken from a git tag
VersionCode -  number of a commit

Reply:
+1 we can do this in another PR